using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,166 @@
+= Red Hat SSO xPaaS Image
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[IMPORTANT]
+This image is currently in https://access.redhat.com/support/offerings/techpreview[Technical Preview] and not intended for production use.
+
+== Overview
+
+Red Hat Single Sign-On (SSO) is an integrated sign-on solution available as a containerized xPaaS image designed for use with OpenShift. This image provides an authentication server for users to centrally login, logout, register, and manage user accounts for web applications, mobile applications, and RESTful web services.
+
+Red Hat offers five SSO Application Templates, each of which deploys two SSO web server pods and a separate pod for the backing database:
+
```

Comment:
I think this sentence might be a little confused. For now, it sounds as though the templates are doing the deploying, when I think it should be OpenShift that is doing the deploying. Maybe:

"Red Hat offers five SSO Application Templates. When deployed using OpenShift, an application consists of two SSO web server pods and a separate pod for the backing database."

or something similar.

Also, does "Application Templates" need to be in caps?